commit 9c63e107efa88786036348ed18e59691c6f4902a (refs/changes/17/3817/1)
Author: dyep49 <dyep49@gmail.com>
Date:   2018-04-13T10:53:00-07:00 (1 year, 6 months ago)
    
    TRITON-322 Change node-workflow-moray-backend package.json to public

diff --git a/package.json b/package.json
index 0af615d..bc9e096 100644
--- a/package.json
+++ b/package.json
@@ -3,7 +3,6 @@
   "name": "wf-moray-backend",
   "description": "A backend for wf built over Moray",
   "version": "1.2.0",
-  "private": true,
   "homepage": "https://github.com/joyent/node-workflow-moray-backend",
   "repository": {
     "type": "git",
